controllers/constants.go

```diff
@@ -1,9 +1,12 @@
 package controllers
 
 const (
-	// KeySelectedNode is an indexing key for the controller
+	// KeySelectedNode is a PVC resource indexing key for the controller
 	KeySelectedNode = "metadata.annotations.selected-node"
 
+	// KeyLogicalVolumeNode is a Logical Volume resource indexing key for the controller
+	KeyLogicalVolumeNode = "spec.nodeName"
```

Comment:
Would you consider making `KeySelectedNode` and `KeyLogicalVolumeNode` package private, since it is not referenced by other packages.